ambuda/static/js/proofer.js

```diff
@@ -82,6 +87,11 @@ export default () => ({
         // initialized. See `init` for details.
         this.imageZoom = settings.imageZoom;
         this.layout = settings.layout || this.layout;
+
+        // Normalize layout value to protect against some recent refactoring.
+        if (![LAYOUT_SIDE_BY_SIDE, LAYOUT_TOP_AND_BOTTOM].includes(this.layout)) {
```

Comment:
Let's move `[LAYOUT_SIDE_BY_SIDE, LAYOUT_TOP_AND_BOTTOM]` to a constant `LAYOUTS` declared at the beginning of the file. This way, we will be more likely to update this array if we add different types of layouts in the future.